src/Symfony/Bridge/PhpUnit/Compat/Framework/BaseTestListener.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Compat\Framework;
+
+if (class_exists(PHPUnit\Framework\BaseTestListener::class)) {
+    abstract class BaseTestListener extends \PHPUnit\Framework\BaseTestListener
```

Comment:
this class (and all other classes here) must be tagged as ``@internal`` so that there are not covered by BC.